src/Simplex/Chat.hs

```diff
@@ -499,22 +497,26 @@ processChatCommand = \case
   APIClearChat (ChatRef cType chatId) -> withUser $ \user@User {userId} -> case cType of
     CTDirect -> do
       ct <- withStore $ \db -> getContact db userId chatId
-      ciIdsAndFileInfo <- withStore' $ \db -> getContactChatItemIdsAndFileInfo db user chatId
-      forM_ ciIdsAndFileInfo $ \(itemId, _, fileInfo_) -> deleteDirectChatItem user ct (itemId, fileInfo_)
-      ct' <- case ciIdsAndFileInfo of
-        [] -> pure ct
-        _ -> do
-          let (_, lastItemTs, _) = last ciIdsAndFileInfo
-          withStore' $ \db -> updateContactTs db user ct lastItemTs
-          pure (ct :: Contact) {updatedAt = lastItemTs}
+      filesInfo <- withStore' $ \db -> getContactFileInfo db user ct
+      maxItemTs_ <- withStore' $ \db -> getContactMaxItemTs db user ct
```

Comment:
This might be better to choose ID for contacts - for example the one that all chat items before this ID are older than specified. This way we will be deleting in the order of the chat, but some older chat items may remain.